Gutenberg/config.yml

```diff
@@ -10,10 +8,12 @@
 # Example:
 #
 #   LOCAL_GUTENBERG=../my-gutenberg-fork bundle exec pod install
-GUTENBERG_CONFIG = {
-  # commit: ''
-  tag: 'v1.101.0'
-}
-
-GITHUB_ORG = 'wordpress-mobile'
-REPO_NAME = 'gutenberg-mobile'
+ref:
+  # This should have either a commit or tag key.
+  # If both are set, tag will take precedence.
+  #
+  # We have automation that reads and updates this file.
+  # Leaving commented keys is discouraged, because as the automation would ignore them and the resulting updates would be noisy
+  tag: v1.101.0
```

Comment:
Is this note going to create confusion down the track?

Alternatively, we could always specify both keys and:

1. Leave the unused one empty / `nil`
2. Maintain the `tag` before `commit` rule

Keen to hear what you think.

Reply:
I think the noted approach should be fine. 👍🏻